_posts/2018-10-16-newpipe-0.14.2-released.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: NewPipe 0.14.2 released, introducing grid layout for tablets
+short: "NewPipe 0.14.2 released"
+date: 2018-10-16 13:00:00 +0200
+categories: [release, pinned]
+author: TobiGr
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+We have just released NewPipe 0.14.2. Such as NewPipe 0.14.0 this version mostly brings UI and UX improvements
+which include new layouts for tablets and an additional linear layout in "Downloads".
+
+Furthermore, the subscriptions page got a small upgrade, several bugs were fixed and new translations added.
+<!-- more -->
+
+## New tablet UI
+
+[@nv95](https://github.com/nv95) brings a completely new feeling to NewPipe on tablets: he implemented grid layouts for video lists like the search, a channel's videos or the trending page. He took a look at the subscriptions page, too:
+
+<img class="no-flow vertical" src="/img/screenshots/shot_subscriptions_tablet_dark.png"/>
+
+On top of that, he also made the video detail view responsive. This means, next and recommended videos will be displayed at the right side of your screen if your device is large enough.
+
+<img class="no-flow vertical" src="/img/screenshots/shot_video_detail_tablet_dark.png"/>
+
+[@jludden](https://github.com/jludden) implemented an additional list layout for the downloads. This might help with these videos which titles have the same beginnings.
+<!-- TODO: linear layout download image -->
+<img class="no-flow" src="/img/screenshots/shot_05.png"/>
+
+## Improved subscription handling
+
+This version brings also interesting changes to the subscription page. [@somethingwieredhere](https://github.com/somethingweiredhere) added a new longpress menu to it. From within the menu you can share and delete a subscription. This is helpful whenever a channel was deleted or renamed by YouTube. So if you have had dead channels before, it is now time to clean them up!
```

Comment:
```suggestion
This version brings also interesting changes to the subscription page. [@somethingwieredhere](https://github.com/somethingweiredhere) added a new longpress menu to it. From within the menu you can share and delete a subscription. This can be really useful whenever a channel was deleted or renamed by YouTube. So, if you have had dead channels before, it is now time to clean them up!
```